src/components/TokenImg.tsx

```diff
@@ -52,4 +52,6 @@ export default styled(TokenImg)<{ size?: number }>`
   border-radius: 100%;
   height: ${({ size }) => size || 1}em;
   width: ${({ size }) => size || 1}em;
+
+  ${({ unapproved }) => unapproved && 'filter: grayscale(100%) opacity(60%)'}
```

Comment:
use `'saturate(0) opacity(0.4)'` instead (I may have the opacity value wrong, but this is what is already in the code). Desaturation is the stated goal, so `saturate` is more semantic than `grayscale`.